shaders/GafferOSL/ObjectProcessing.h

```diff
@@ -108,9 +108,21 @@ closure color outColor( string name, color value )
 	return debug( name, "type", "color", "value", color( value ) );
 }
 
+// Custom closure used by ShadingEngine to detect shaders that will
+// deform an object.
+closure color deformation( string name ) [[ int builtin = 1 ]];
```

Comment:
If we decide width counts as a deformation as per my prior comment, then we need to account for it in `ShadingEngine` as well.